### PR TITLE
modify acgan

### PR DESCRIPTION
**Pull request by San-ctuary**
The CrossEntropyLoss contains softmax(),so it does not need to add softmax() before the loss funciton

### CODE REVIEW

**Comment by lanlanwei** [comment]
您好，您的来信我已收到，感谢来信，祝天天好心情。

**Comment by baicaiPCX** [comment]
您好，您的邮件小皮已经收到啦！！！
